documentation/tests/test_documentation_rendering.py

```diff
@@ -3,28 +3,52 @@
 from curricula.factories import CurriculumFactory, UnitFactory
 from lessons.factories import LessonFactory
 
-from documentation.factories import IDEFactory, BlockFactory
+from documentation.factories import IDEFactory, BlockFactory, CategoryFactory, MapFactory
 
 
 class DocumentationRenderingTestCase(TestCase):
     def setUp(self):
-        self.myIDE = IDEFactory(slug="myIDE")
-        self.myBlock = BlockFactory(slug="myBlock", title="My Block", parent_ide=self.myIDE, parent=self.myIDE)
+        self.myIDE = IDEFactory(slug="mylab")
+        self.myCategory = CategoryFactory(name="myCategory", parent_ide=self.myIDE)
+        self.myBlock = BlockFactory(slug="myBlock", title="myBlock", parent_ide=self.myIDE, parent=self.myIDE, parent_cat=self.myCategory)
 
         self.test_curriculum = CurriculumFactory(slug="test-curriculum")
         self.test_unit = UnitFactory(parent=self.test_curriculum, slug="test-unit")
 
         self.test_lesson = LessonFactory(title="My Lesson", parent=self.test_unit)
         self.test_lesson.blocks.add(self.myBlock)
 
+        self.myConcept = MapFactory(slug="concepts", title="Concepts")
+        self.myMap = MapFactory(slug="concepts/myMap", title="My Map", parent=self.myConcept)
+
     def test_render_course_blocks_page(self):
         response = self.client.get('/test-curriculum/code/')
         self.assertEqual(response.status_code, 200)
         self.assertIn('My Lesson', response.content)
-        self.assertIn('My Block', response.content)
+        self.assertIn('myBlock', response.content)
```

Comment:
it looks like the "My Map" title is working in the test below. Is it possible to change this back to "My Block" so that it's clear this is matching on the title and not the slug?